File: 백준/Bronze/5073. 삼각형과 세 변/삼각형과 세 변.py
while(True):
  a, b, c = map(int, input().split())
  if a == b == c == 0:
    break
  
  if a < b + c and b < a + c and c < a + b:
    if(a == b == c):
      print("Equilateral")
    elif(a == b or b == c or a == c):
      print("Isosceles")
    else:
      print("Scalene")
  else:
    print("Invalid")

File: 백준/Bronze/5073. 삼각형과 세 변/삼각형과 세 변.c
#include <stdio.h>

int main(void) {  
   int a, b, c;
   while(1){
      scanf("%d %d %d", &a, &b, &c);
      if(a == 0 && b == 0 && c == 0) break;

      if(a + b > c && a + c > b && b + c > a){
         if(a == b && b == c){
            printf("Equilateral\n");
         }
         else if(a == b || a == c || b == c){
            printf("Isosceles\n");
         }
         else{
            printf("Scalene\n");
         }
      }
      else{
         printf("Invalid\n");
      }
   }
}